# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eq-xmms/slider.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- eq-xmms-0.6_orig/src/equalizer.c	2004-04-03 00:28:04.000000000 +0200
+++ eq-xmms-0.6/src/equalizer.c	2005-04-10 17:35:01.389577720 +0200
@@ -755,7 +755,7 @@
     EQequalizer_volume = EQcreate_hslider(&EQequalizer_wlist, EQequalizer_bg, EQequalizer_gc, 61, 4, 97, 8, 1, 30, 1, 30, 3, 7, 4, 61, 0, 94, EQequalizer_volume_frame_cb, EQequalizer_volume_motion_cb, EQequalizer_volume_release_cb, SKIN_EQSHADE);
     EQequalizer_balance = EQcreate_hslider(&EQequalizer_wlist, EQequalizer_bg, EQequalizer_gc, 164, 4, 42, 8, 11, 30, 11, 30, 3, 7, 4, 164, 0, 39, EQequalizer_balance_frame_cb, EQequalizer_balance_motion_cb, EQequalizer_balance_release_cb, SKIN_EQSHADE);
     EQequalizer_set_shade(eqcfg.shaded);
-    EQequalizer_set_shade_slider();
+//    EQequalizer_set_shade_slider();
     // end of shade elements
 
     tpix = EQcreate_pbutton(&EQequalizer_wlist, EQequalizer_bg, EQequalizer_gc, 42, 33, 26, 10, 0, 11, 27, 11, EQequalizer_quickhi_pushed, SKIN_EQQUICK);
